File: TOI-Zero_68/A3/021/021.py
import math

n, k = map(int, input().split())
a = [int(input()) for _ in range(n)]
a.sort()

x = k * a[0]
count = 1  # Start with 1 since we're checking from index 1

for i in range(1, n):
    if math.ceil(x / a[i]) == k:
        count += 1

print(count) 

File: TOI-Zero_68/A3/021/021.cpp
#include<bits/stdc++.h>
using namespace std;

#define ll long long

int main(){
	ios::sync_with_stdio(0);
	cin.tie(0);
	int n,k;
    cin >> n >> k;
    ll a[n];
    for(int i=0;i<n;i++){
    	cin >> a[i];
	}
    sort(a,a+n);
    ll x=k*a[0],count=1;
    for(int i=1;i<n;i++){
        count+=(ceil((double)x/a[i])==k);
    }
    cout << count;
}
